messenger.go

```diff
@@ -116,6 +123,65 @@ func (m *Messenger) ProfileByID(id int64) (Profile, error) {
 	return p, err
 }
 
+// GreetingSetting sends settings for greeting
+func (m *Messenger) GreetingSetting(text string) error {
+	d := GreetingSetting{
+		SettingType: "greeting",
+		Greeting: GreetingInfo{
+			Text: text,
+		},
+	}
+
+	data, err := json.Marshal(d)
+	if err != nil {
+		return nil
```

Comment:
This bug come from here https://github.com/paked/messenger/blob/master/response.go#L42. I'll fix them both.